src/components/mols/SquadMembers/index.tsx

```diff
@@ -0,0 +1,25 @@
+import type { ReactNode } from 'react';
+import React from 'react';
+
+import { Card, IconButton } from '@/components/atoms';
+import { Typography } from '@/components/atoms/Typography';
+
+export type SquadMemberProps = {
+  children?: ReactNode;
```

Comment:
Instead of receiving a children let's add the`users: User[]` prop. Inside the SquadMember loop through these users and call the Userbar passing each user data.

Note: the User type of data must be the same as the data received by UserBar